scripts/xdist/find_order_dependent_test_failures.py

```diff
@@ -143,9 +153,15 @@ def _run_tests_and_check_for_failures(output_file_name):
     """
     global verbose_option
     pytest_command = _get_pytest_command(output_file_name)
+    # if verbose_option:
+    print(pytest_command)
     test_output = os.popen(pytest_command).read()
     if verbose_option:
         print(test_output)
+    # errors_search = re.search(r'=== \d+ warnings, (\d+) errors', test_output)
+    # if bool(errors_search) and int(errors_search.group(1)) > 0:
+    #     print(f"There is a problem running tests. {int(errors_search.group(1))} errors found when running tests.")
+    #     exit(1)
```

Comment:
**Author Note:** Before making any fixes, if you run the test `console.txt` against cms, you will see errors that occur during test collection, before the tests get to run. We should notify of this problem. This conditional works fine. It may be as simple as swapping this print/exit with a raised exception, so any open file can properly be closed.

```
python scripts/xdist/find_order_dependent_test_failures.py --log-file console.txt --test-suite cms-unit
```